app/views/shared/_records_table.html.erb

```diff
@@ -25,6 +25,7 @@
 
     <div class="results-table-container">
       <div class="table-scroll">
+        My React Component
```

Comment:
Remove this, test string left in from me.

Reply:
removed